app-shells/autojump/autojump-22.2.4-r2.ebuild

```diff
@@ -42,21 +38,17 @@ src_install() {
 	insinto /etc/profile.d
 	doins bin/${PN}.sh
 
-	if use bash-completion ; then
-		doins bin/${PN}.bash
-	fi
-
-	if use zsh-completion ; then
-		doins bin/${PN}.zsh
-		insinto /usr/share/zsh/site-functions
-		doins bin/_j
-	fi
+	newbashcomp bin/${PN}.bash ${PN}
+	insinto /usr/share/"${PN}"/
```

Comment:
PN is not allowed to have whitespaces anyway (unless you overwrite it with funny stuff), so just remove all quotes